src/shapes/active_selection.class.js

```diff
@@ -24,58 +24,42 @@
      */
     type: 'activeSelection',
 
+    /**
+     * @override
+     */
+    layout: 'fit-content',
+
+    /**
+     * @override
+     */
+    subTargetCheck: false,
+
+    /**
+     * @override
+     */
+    interactive: false,
```

Comment:
subTargetCheck and being able to select and modify sub objects are 2 distinct use cases.

Reply:
I can commit this to v6! if you want, thought of it too

Reply:
we will need to rebase on this once we bring another piece on master. So if this work out it will go in naturally.

Reply:
Why do you think it's 2 distinct use cases?

Reply:
Alright I think I understand.
e.g. ActiveSelection should not be interactive be should have subTargetCheck set to true?

Reply:
well once it did not need it. Now it seems to need it.
But apart from that that is minor, you may want to fire events for nested targets, but not need to select them.
We could argue that events ( mouse up, mouse down ) could fire 100% of the time since this is a breaking version, but i m not sure that would be an issue for some application that added on the mousedown/mouseup events logic that won't work on group. ( they can still filter it out checking if an object has group )

Reply:
Didn't understand